querybook/webapp/ui/SimpleReactSelect/SimpleReactSelect.tsx

```diff
@@ -59,13 +67,23 @@ export function SimpleReactSelect<T>({
         [options]
     );
 
-    const selectedOption = useMemo(
-        () => computedOptions.find((option) => option.value === value),
-        [computedOptions, value]
-    );
+    const selectedOption = useMemo(() => {
+        if (isMulti) {
```

Comment:
this breaks the point of use SimpleReactSelect, why not just use native React Select?